Makefile

```diff
@@ -92,7 +92,7 @@ bin/promtool: bin/promtool-${PROMTOOL_VERSION}/promtool
 
 bin/gator-${GATOR_VERSION}/gator:
 	mkdir -p bin/gator-${GATOR_VERSION}
-	curl -sSfL https://github.com/open-policy-agent/gatekeeper/releases/download/v${GATOR_VERSION}/gator-v${GATOR_VERSION}-${GOHOSTOS}-${GOHOSTARCH}.tar.gz | tar zxf - -C bin/gator-${GATOR_VERSION} --strip=1 gator
+	curl -sSfL https://github.com/open-policy-agent/gatekeeper/releases/download/v${GATOR_VERSION}/gator-v${GATOR_VERSION}-${GOHOSTOS}-${GOHOSTARCH}.tar.gz -o - | tar zxf - -C bin/gator-${GATOR_VERSION} gator
```

Comment:
`curl` outputs to stdout by default, explicit option is unneccesary.
Here removed `--strip=1` is what fixes download, cause gator wasn't nested it a directory as promtool was.

Reply:
curl work differs on different distros. On my mac this was a problem.